src/index.ts

```diff
@@ -16,6 +16,7 @@ export type {
 } from './hooks/use-toast-messages';
 
 // Utils
+export * from './util/date-and-time';
```

Comment:
Actually, now that I look at the exports of `date-and-time`, some of them may not have an obvious meaning if you just import from the root:

```
import { clearFormatters } from '@hypothesis/frontend-shared'
```

Requiring an import from a submodule would make this easier to follow:

```
import { clearFormatters } from '@hypothesis/frontend-shared/date-and-time'
```

Reply:
@acelaya - Ah, I see you didn't export this. Presumably we don't need this downstream because it is only used for tests in the package itself? That's another reason to favor explicit exports.

Reply:
Ouch, I didn't see this commit before merging the PR.

I considered providing a dedicated entry point for date functions (see [slack thread](https://hypothes-is.slack.com/archives/C1M8NH76X/p1727789002774409)), but ended up not doing it for simplicity.

However, what I have done is not exposing `clearFormatters`, as this is relevant for tests only, and it can be imported from the actual module in that case.

The rest of symbols have meaning on their own, but we can still provide a dedicated endpoint if we think that makes more sense.